crates/cairo-profiler/src/main.rs

```diff
@@ -18,7 +18,7 @@ mod trace_reader;
 
 #[derive(Parser, Debug)]
 #[command(version)]
-#[clap(name = "cairo-profiler")]
+#[clap(name = "cairo-cairo-profiler")]
```

Comment:
?

Reply:
It is not there, you were looking at wrong diff